src/Normalizer/JsonldContextTrait.php

```diff
@@ -1,45 +1,38 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: dpino
- * Date: 9/1/16
- * Time: 9:33 PM
- */
 
 namespace Drupal\jsonld\Normalizer;
 
-  /**
+/**
    * JSON-LD context helper trait.
    *
    * @author Diego Pino Navarro
    *
    * @internal
    */
-trait JsonLdContextTrait
-{
+trait JsonLdContextTrait {
```

Comment:
@dannylamb i just revived it for what you asked @Natkeeran to-do. Can remove of course.